tools/lint/cmd_data.c

```diff
@@ -417,6 +453,34 @@ evaluate_xpath(const struct lyd_node *tree, const char *xpath)
     return 0;
 }
 
+int
+parse_ext_string(char *extension_name, struct yl_opt *yo)
```

Comment:
In this case modifying `extension_name` (not a good param name) should be fine but in general is not a good practice so please change to `const char *` and parse the fields without the use of `strtok()`.